packages/@react-aria/tag/src/useTag.ts

```diff
@@ -66,11 +66,18 @@ export function useTag<T>(props: TagProps<T>, state: TagGroupState<T>, ref: RefO
     }
   };
 
+  let modality: string = useInteractionModality();
+  if (modality === 'virtual' &&  (typeof window !== 'undefined' && 'ontouchstart' in window)) {
+    modality = 'touch';
+  }
+  let description = allowsRemoving && (modality === 'keyboard' || modality === 'virtual') ? stringFormatter.format('removeDescription') : '';
+  let descProps = useDescription(description);
+
   isFocused = isFocused || state.selectionManager.focusedKey === item.key;
   let domProps = filterDOMProps(props);
   return {
     clearButtonProps: {
-      'aria-label': removeString,
+      'aria-label': stringFormatter.format('removeButtonLabel', {label: item.textValue}),
```

Comment:
I don't think you should need the label in the `aria-label` because you already have an `aria-labelledby` below that points to the label element in addition to the button. Right now I'm hearing the label announced twice. So just the word "Remove" should be enough here?